pkg/jamf/client.go

```diff
@@ -76,7 +76,7 @@ func (c *Client) CreateBearerToken(
 	request.SetBasicAuth(username, password)
 
 	var target TokenResponse
-	response, err := c.wrapper.Do(request, uhttp.WithJSONResponse(target))
+	response, err := c.wrapper.Do(request, uhttp.WithJSONResponse(&target))
```

Comment:
Could this be the problem?

Reply:
> Could this be the problem?

AFAIK, not the case, `target` wouldn't be nil when the pointer reference is being asked, so I don't think this would cause a `panic`.

Reply:
oh, nvm my message lol.